dev-qt/qt5/qt5-5.6.1.recipe

```diff
@@ -9,7 +9,7 @@ LICENSE="GNU LGPL v2"
 
 SOURCE_URI="https://download.qt.io/official_releases/qt/5.6/$portVersion/single/qt-everywhere-opensource-src-$portVersion.tar.xz"
```

Comment:
There were 2 ways to fix this recipe:
* change the 64 characters of the CHECKSUM_SHA256.
* **_or_** change a single character: tar.**x**z → tar.**g**z

Since you had created the recipe with the SHA256 of the **gz**ipped tarball, it was clear to me that you had downloaded that tarball.

Now, if you want to switch back to the tar.xz you know you don't need anyone's permission.